src/Murder.Editor/Data/Graphics/RectpackSharp.cs

```diff
@@ -0,0 +1,769 @@
+        /// <returns>The value that should be assigned to <see cref="PackingRectangle.SortKey"/>.</returns>
+        private delegate uint GetSortKeyDelegate(in PackingRectangle rectangle);
+
+        /// <summary>The maximum amount of hints that can be specified by a <see cref="PackingHint"/>.</summary>
+        internal const int MaxHintCount = 6;
+
+        public static uint GetArea(in PackingRectangle rectangle) => rectangle.Area;
+        public static uint GetPerimeter(in PackingRectangle rectangle) => rectangle.Perimeter;
+        public static uint GetBiggerSide(in PackingRectangle rectangle) => rectangle.BiggerSide;
+        public static uint GetWidth(in PackingRectangle rectangle) => rectangle.Width;
+        public static uint GetHeight(in PackingRectangle rectangle) => rectangle.Height;
+        public static uint GetPathologicalMultiplier(in PackingRectangle rectangle) => rectangle.PathologicalMultiplier;
+
+        /// <summary>
+        /// Separates a <see cref="PackingHint"/> into the multiple options it contains,
+        /// saving each of those separately onto a <see cref="Span{T}"/>.
+        /// </summary>
+        /// <param name="packingHint">The <see cref="PackingHint"/> to separate.</param>
+        /// <param name="span">The span in which to write the resulting hints. This span's excess will be sliced.</param>
+        public static void GetFlagsFrom(PackingHints packingHint, ref Span<PackingHints> span)
+        {
+            int index = 0;
+            if (packingHint.HasFlag(PackingHints.TryByArea))
+                span[index++] = PackingHints.TryByArea;
+            if (packingHint.HasFlag(PackingHints.TryByPerimeter))
+                span[index++] = PackingHints.TryByPerimeter;
+            if (packingHint.HasFlag(PackingHints.TryByBiggerSide))
+                span[index++] = PackingHints.TryByBiggerSide;
+            if (packingHint.HasFlag(PackingHints.TryByWidth))
+                span[index++] = PackingHints.TryByWidth;
+            if (packingHint.HasFlag(PackingHints.TryByHeight))
+                span[index++] = PackingHints.TryByHeight;
+            if (packingHint.HasFlag(PackingHints.TryByPathologicalMultiplier))
+                span[index++] = PackingHints.TryByPathologicalMultiplier;
+            span = span.Slice(0, index);
+        }
+
+        /// <summary>
+        /// Sorts the given <see cref="PackingRectangle"/> array using the specified <see cref="PackingHint"/>.
+        /// </summary>
+        /// <param name="rectangles">The rectangles to sort.</param>
+        /// <param name="packingHint">The hint to sort by. Must be a single bit value.</param>
+        /// <remarks>
+        /// The <see cref="PackingRectangle.SortKey"/> values will be modified.
+        /// </remarks>
+#if NET5_0_OR_GREATER
+        public static void SortByPackingHint(Span<PackingRectangle> rectangles, PackingHints packingHint)
+#elif NETSTANDARD2_0
```

Comment:
I think we can remove these `#ifdefs`? Since Murder only targets .NET 8.

Reply:
[RectpackSharp](https://github.com/ThomasMiz/RectpackSharp) is referenced from another library's implementation, and I'm wondering if I should reference its nuget package or would it be better to put it in a separate file?

Reply:
If you don't want to add nuget package references, I think you can remove those "#ifdefs"